dev-tools/packaging/package_test.go

```diff
@@ -382,6 +384,31 @@ func checkDockerEntryPoint(t *testing.T, p *packageFile, info *dockerInfo) {
 	})
 }
 
+func checkDockerLabels(t *testing.T, info *dockerInfo, file string) {
+	if vendor := info.Config.Labels["org.label-schema.vendor"]; vendor != "Elastic" {
+		t.Errorf("unexpected vendor label: %s", vendor)
+	}
```

Comment:
Should we provide a way to skip Elastic-specific tests for community beats?

Reply:
@graphaelli wdyt about this?

Reply:
I agree we should provide a mechanism for this, I just hadn't come up with a good way short of wiring through a command line flag for it.  I've pushed up a commit to only perform this check if the vendor label is `Elastic`, which the user should customize.  This of course relies on that label working so it's not my favorite solution but should be good enough to close this out finally - ok?

Reply:
:+1: